File: src/app/gkill/api/req_res/add_kc_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type AddKCResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	AddedKC *reps.KC `json:"added_kc"`

	AddedKCKyou *reps.Kyou `json:"added_kc_kyou"`
}

File: src/classes/api/req_res/add-kc-response.ts
'use strict'

import { KC } from '@/classes/datas/kc'
import { GkillAPIResponse } from '../gkill-api-response'
import { Kyou } from '@/classes/datas/kyou'

export class AddKCResponse extends GkillAPIResponse {

    added_kc: KC

    added_kc_kyou: Kyou

    constructor() {
        super()
        this.added_kc = new KC()
        this.added_kc_kyou = new Kyou()
    }

}


